CONTRIBUTING.md

```diff
@@ -1,60 +1,202 @@
 # Contributing
 
-## Code Quality
+1. [Getting Started](#getting-started)
+   - [Discussing Proprietary Features](#discussing-proprietary-features)
+   - [Setup](#setup)
+1. [Developing Components](#developing-components)
+   - [Tools Used](#tools-used)
+   - [Structure](#structure)
+   - [Quality Checklist](#quality-checklist)
+   - [Server-side Rendering Compatibility](#server-side-rendering-compatibility)
+   - [Linting](#linting)
+   - [Icons](#icons)
+   - [Commit Messages](#commit-messages)
+   - [Pull Requests](#pull-requests)
+   - [Testing Strategies](#testing-strategies)
+   - [Unit Tests](#unit-tests)
+   - [Accessibility Testing](#accessibility-testing)
+1. [Documentation](#documentation)
+1. [Breaking Changes](#breaking-changes)
+   - [Components](#components)
+   - [Styles](#styles)
+   - [Deprecating](#deprectating)
+   - [Removal](#removal)
+
+## Getting Started
+
+Cauldron is Deque's design system / pattern library for building accessible ui as part of our products as Deque. Cauldron consists of several parts, [react](./packages/react), [styles](./packages/styles), and [documentation](./docs).
```

Comment:
```suggestion
Cauldron is Deque's design system/pattern library for building accessible UI as part of our products as Deque. Cauldron consists of several parts, [react](./packages/react), [styles](./packages/styles), and [documentation](./docs).
```

Reply:
"design system" and "pattern library" are open compounds, so I believe the proper writing style is to include a space between the slash.

Reply:
Oh yeah, you're right. Grammarly wanted to make it this way. 😄